sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -35,7 +36,8 @@
  * @see Configuration
  */
 public class ClientLogger {
-    private final Logger logger;
+    private Logger logger;
```

Comment:
Why is this no longer final? Instead load the SLF4J binded instance into a temp variable then set this based on that being a NOPLogger.

Reply:
Revert back to final